submit-api/src/submit_api/services/note.py

```diff
@@ -0,0 +1,41 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Service for internal staff document service."""
+from submit_api.exceptions import ResourceNotFoundError
+from submit_api.models import Item
+from submit_api.models.note import Note as NoteModel
+from submit_api.utils.token_info import TokenInfo
+
+
+class NoteService:
```

Comment:
In theory, we could have notes for any entity. That's why I would change any naming from "Notes" -> "SubmissionNote" or "SubmissionItemNote". And that goes everywhere (the resource, the service, the model, the schema etc.) Just so we don't get caught in an awkward position when we have to add a Note  to some other entity like Package per se.